content/en/spec/activitypub.md

```diff
@@ -53,21 +53,21 @@ Some other Object types are converted as best as possible:
 - Video
 - Event
 
-The transformer uses `content` if available, or `name` if not, in order to generate status text. The `url` will be appended. The `summary` property will be used as the CW text. The `icon` will be used as a thumbnail.
+The transformer uses `content` to generate status text. If `content` is not available, it uses `name` instead. The `url` will be appended. The `summary` property will be used as the Content Warning (CW) text. The `icon` will be used as a thumbnail.
 
-### HTML sanitization {#sanitization}
+### HTML sanitisation {#sanitization}
```

Comment:
same concern about en-uk vs en-us, except this also makes the fragment id inconsistent